Gemfile

```diff
@@ -4,11 +4,7 @@ source 'https://rubygems.org'
 gemspec
 
 group :local_development do
-  gem 'pry'
   platforms :mri do
-    if RUBY_VERSION >= '2.0.0'
-      gem 'byebug'
-      gem 'pry-byebug'
-    end
+    gem 'pry-byebug'
```

Comment:
No more need for the guard regarding 2.0.0 - with the next release coming from "develop" we'll drop ruby support for 1.9 so this is obsolete.